### DISCUSSION TITLE
Display output outside/without the editor

### DISCUSSION THREAD

**Discussion by J87NL** [jeffreyvr/dropblockeditor#27] [General]
Hi Jeffrey,

Cool project! We've got the DropBlockEditor up and running but we wonder: how can we render and display the output (without the editor) on a webpage? Did we miss something in the docs?

Kind regards,
Johan.

**Comment by jeffreyvr** [comment]
Thanks!

The example button gives you an idea on how you can do that:
https://dropblockeditor.com/docs/beta/making-buttons

But assuming you have the resulting JSON stored somewhere, you can:

```php
$output = Parse::execute([
   'activeBlocks' => json_decode($activeBlocksJson, true),
   'base' => 'base template you are using',
   'context' => 'rendered',
   'parsers' => [Jeffreyvr\DropBlockEditor\Parsers\Html::class],
]);
```

**Comment by J87NL** [comment]
Hi Jeffrey, thanks for your fast reply, this works like a charm!
